AllReadyApp/Web-App/AllReady/package.json

```diff
@@ -12,6 +12,7 @@
     "gulp-cssmin": "0.1.7",
     "gulp-sourcemaps": "1.6.0",
     "gulp-typescript": "3.1.5",
+    "natives": "1.1.6",
```

Comment:
I read some of the notes and warnings about this package behaving differently on different versions of NodeJS.

Reply:
I ran across the same but I think the best long term solution is simply move to Gulp 4.0.  I’ve got a separate PR for that in the pipeline. Decided it was better to piecemeal out things.

There are a lot of packages.  Not sure if they are all being used either.  NPM audit shows a bunch of stuff well above my pay grade :)